BlogTemplate/Pages/_LoginPartial.cshtml

```diff
@@ -6,20 +6,11 @@
 @if (SignInManager.IsSignedIn(User))
 {
     <form asp-controller="Account" asp-action="Logout" method="post" id="logoutForm" class="navbar-right">
-        <ul class="nav navbar-nav navbar-right">
-            <li>
-                <a asp-page="/Account/Manage/Index" title="Manage">Hello @UserManager.GetUserName(User)!</a>
-            </li>
-            <li>
+                <a asp-page="/Account/Manage/Index" title="Manage">Owner Settings</a>
                 <button type="submit" class="btn btn-link navbar-btn navbar-link">Log out</button>
-            </li>
-        </ul>
     </form>
 }
 else
 {
-    <ul class="nav navbar-nav navbar-right">
-        <li><a asp-page="/Account/Register">Register</a></li>
-        <li><a asp-page="/Account/Login">Log in</a></li>
-    </ul>
+       <a asp-page="/Account/Login">Log in</a>

```

Comment:
oh, did you mean that going to Login still shows the login form?  In that case, yes, it would make better sense to redirect the user too.